docs/awesome-pets/papers/applications/ppml/fl/survey.md

```diff
@@ -4,6 +4,7 @@
 - [Federated Learning in Mobile Edge Networks: A Comprehensive Survey](https://arxiv.org/pdf/1909.11875.pdf)
 - [Advances and Open Problems in Federated Learning](https://arxiv.org/pdf/1912.04977.pdf)
 - [Federated Learning: Challenges, Methods, and Future Directions](https://arxiv.org/pdf/1908.07873.pdf)
+- [Federated Learning for Internet of Things: Applications, Challenges, and Opportunities](https://arxiv.org/pdf/2111.07494.pdf)
```

Comment:
Please add the conference or journal  which this article has been accepted

Reply:
Zhang, Tuo et al. “Federated Learning for the Internet of Things: Applications, Challenges, and Opportunities.” IEEE Internet of Things Magazine 5 (2021): 24-29.